src/components/plots/MiniPlot.jsx

```diff
@@ -75,4 +74,4 @@ MiniPlot.propTypes = {
   actions: PropTypes.bool.isRequired,
 };
 
-export default MiniPlot;
+export default React.memo(MiniPlot);
```

Comment:
`MiniPlot` needs to be memoized. It is often loaded from a big component, so every rerender of the main component will cause a rerender of the miniplot as well. `React.memo` makes sure (at the expense of some performance degradation) that the component only renders from scratch when the props change. Since this component uses no internal state, can be quite heavy to render from scratch, and is fully described by the props, it is a good idea to use it here.